js/app/restaurant/restaurant.provider.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Provides data from a json feed provided by Site Executive Structured Content
+ */
+
+'use strict';
+
+((app) => {
+	const restaurantProvider = (constants, RestaurantModel, $http, $q) => {
+		const formatCategories = categories => categories.map(category => category.LABEL);
+		const formatPhoneNumber = number => number.toString().replace(/(\d{3})(\d{3})(\d{4})/, '$1-$2-$3');
+
+		const mapRestaurants = (structuredContentData) => {
+			const mappedRestaurants = [];
+
+			structuredContentData.forEach((mappedRestaurant) => {
+				const { website, logo } = mappedRestaurant;
```

Comment:
I'm not familiar with this syntax.  Is `mappedRestaurant` a 2-member array?

Reply:
Object destructuring a nice little feature in javascript, preferred by airbnb eslint config

Reply:
I did adjust mappedRestaurant name, because i thought it was a little clunky